README.rst

```diff
@@ -10,19 +10,21 @@ pyfar
 .. image:: https://circleci.com/gh/pyfar/pyfar.svg?style=shield
     :target: https://circleci.com/gh/pyfar/pyfar
 .. image:: https://mybinder.org/badge_logo.svg
-    :target: https://mybinder.org/v2/gh/pyfar/gallery/main?filepath=docs/gallery
+    :target: https://mybinder.org/v2/gh/pyfar/gallery/main?labpath=docs/gallery/interactive/pyfar_introduction.ipynb
 
 
 The python package for acoustics research (pyfar) offers classes to store
 audio data, filters, coordinates, and orientations. It also contains common
-functions for digital audio signal processing.
+functions for digital audio signal processing and plotting audio signals.
 
 Getting Started
 ===============
 
-Check out `pyfar.org`_ for a tour through the pyfar
-universe, including complete documentation of this package and
-the other packages.
+The `pyfar workshop`_ gives an overview of the most important pyfar
+functionality and is a good starting point. It is part of the
+`pyfar example gallery`_ that also contains more specific and in-depth
+examples. The  `pyfar documentation`_ gives a detailed and complete
+overview of pyfar. All these information are available from `pyfar.org`_.
```

Comment:
```suggestion
The `pyfar workshop`_ gives an overview of the most important pyfar
functionality and is a good starting point. It is part of the
`pyfar example gallery`_ that also contains more specific and in-depth
examples. All gallery example notebooks can be executed interactively without a local installation by clicking the mybinder.org button on the respective example. The  `pyfar documentation`_ gives a detailed and complete
overview of pyfar. All these information are available from `pyfar.org`_.
```
even if we keep it, i would add the info about the interactive notebooks from Marco.

Reply:
good point - added this information